hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/TokenTupleUtils.java

```diff
@@ -62,7 +65,7 @@ public static Tuple expiryTupleFor(@NonNull final Token token) {
         return Tuple.of(
                 token.expirationSecond(),
                 headlongAddressOf(token.autoRenewAccountIdOrElse(ZERO_ACCOUNT_ID)),
-                (token.autoRenewSeconds() > 0 ? token.autoRenewSeconds() : 0));
```

Comment:
I don't see anywhere in the code where non-negativity of autoRenewSeconds is enforced, if not here. Maybe it doesn't belong here but instead over at Token.Builder.autoRenewSeconds (which unfortunately is generated by PBJ).

Anyway, I may have missed it.

Reply:
`TokenCreateHandler` should not be creating `Token`'s with negative `autoRenewSeconds`.